plugins/JetConstituentTableProducer.cc

```diff
@@ -48,19 +48,23 @@ class JetConstituentTableProducer : public edm::stream::EDProducer<> {
   //const std::string name_;
   const std::string name_;
   const std::string nameSV_;
+	const std::string nameMu_;
```

Comment:
[Style] Indentation level seems to vary for the newly added lines, compared to the surrounding statements